manifests/t/Teclib'/GLPIAgent1/7/2/1.7.2/Teclib'.GLPIAgent1.7.2.locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Teclib'.GLPIAgent1.7.2
+PackageVersion: 1.7.2
+PackageLocale: en-US
+Publisher: Teclib'
+PackageName: GLPI Agent 1.7.2

```

```suggestion
PackageName: GLPI Agent
```
Having the version here will cause matching errors in the future